stubs/taint/core_privacy_security/django-filesystem-sinks.pysa

```diff
@@ -0,0 +1,10 @@
+def django.core.files.storage.FileSystemStorage.__init__(self, location: TaintSink[FileSystem_ReadWrite], base_url: TaintSink[FileSystem_ReadWrite], file_permissions_mode, directory_permissions_mode): ...
```

Comment:
Unused parameters aren't needed. We usually try to omit them because it reduces the chances of our models breaking due to a parameter rename
```suggestion
def django.core.files.storage.FileSystemStorage.__init__(location: TaintSink[FileSystem_ReadWrite], base_url: TaintSink[FileSystem_ReadWrite]): ...
```